### ISSUE TITLE
WPS export payload requests must include mimeType inside wps:Output 

### ISSUE THREAD

**Issue by giohappy** [geosolutions-it/MapStore2#7536]
WPS export requests from MapStore generate a payload like the following:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<wps:Execute xmlns:wps="http://www.opengis.net/wps/1.0.0" xmlns="http://www.opengis.net/wps/1.0.0" xmlns:dwn="http://geoserver.org/wps/download" xmlns:gml="http://www.opengis.net/gml" xmlns:ogc="http://www.opengis.net/ogc" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:wcs="http://www.opengis.net/wcs/1.1.1" xmlns:wfs="http://www.opengis.net/wfs" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0.0" service="WPS" xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsAll.xsd">
   <ows:Identifier>gs:Download</ows:Identifier>
   <wps:DataInputs>
      <wps:Input>
         <ows:Identifier>layerName</ows:Identifier>
         <wps:Data>
            <wps:LiteralData>geonode:PAT_PO</wps:LiteralData>
         </wps:Data>
      </wps:Input>
      <wps:Input>
         <ows:Identifier>outputFormat</ows:Identifier>
         <wps:Data>
            <wps:LiteralData>application/geopackage+sqlite3</wps:LiteralData>
         </wps:Data>
      </wps:Input>
      <wps:Input>
         <ows:Identifier>cropToROI</ows:Identifier>
         <wps:Data>
            <wps:LiteralData>false</wps:LiteralData>
         </wps:Data>
      </wps:Input>
   </wps:DataInputs>
   <wps:ResponseForm>
      <wps:ResponseDocument storeExecuteResponse="true" status="true">
         <wps:Output asReference="true">
            <ows:Identifier>result</ows:Identifier>
         </wps:Output>
      </wps:ResponseDocument>
   </wps:ResponseForm>
</wps:Execute>
```
Although the outpuFormat is set to `application/geopackage+sqlite3` the wps:Output doesn specify the mimeType and Geoserver will return an href with mimeType `image/tiff` (I guess this is the default).  Geoserver [docs reports that the mimeType is expected to be set inse the wps:Output](https://docs.geoserver.org/latest/en/user/extensions/wps-download/rawDownload.html#output-format-and-response-mime-types).

The final effect is not immediately visible, since the final request to obtain the file output returns a Content-Disposition header, and the filename has the expected extension. However the wrong mimeType can cause troubles when the request is managed differently or routed by proxies (as in the case of GeoNode).

The solution is simple: include the mimeType inside wps:Output as in the follwing excerpt:

`<wps:Output asReference="true" mimeType="application/geopackage+sqlite3">`

**Comment by tdipisa** [comment]
Thank you @giohappy
@allyoucanmap I think this is an issue that @DavidQuartz or @tonym3nsa can check next days. Fell free to assign one of them.

**Comment by simboss** [comment]
@tdipisa @allyoucanmap I need this fixed quickly on 3.3.x.
Thanks...

**Comment by tdipisa** [comment]
> @tdipisa @allyoucanmap I need this fixed quickly on 3.3.x. Thanks...

If so @giohappy can schedule it with @allyoucanmap accordingly.

**Comment by tdipisa** [comment]
@allyoucanmap please sort this this milestone, thank you.
